steps/infisical-secrets/0.0.2/step.yml

```diff
@@ -0,0 +1,65 @@
+title: Infisical Secrets
+summary: |
+  Pull secrets from an Infisical secrets repository
+description: |
+  Use this step to access your cloud or self hosted Infisical secrets and inject them into your build pipeline.
+website: https://github.com/CodeFishStudio/bitrise-step-infisical-secrets
```

Comment:
This URL seems to be invalid, I guess the Step repository was renamed to `https://github.com/CodeFishStudio/infisical-bitrise-step`.

Please update the `website`, `source_code_url`, and `support_url` fields.